src/decoder.c

```diff
@@ -239,6 +239,26 @@ int nanocbor_get_tag(nanocbor_value_t *cvalue, uint32_t *tag)
     return res;
 }
 
+int nanocbor_get_decimal_frac(nanocbor_value_t *cvalue, int32_t *e, int32_t *m)
+{
+    int res = NANOCBOR_NOT_FOUND;
+    uint32_t tag;
+    nanocbor_get_tag(cvalue, &tag);
+    if (tag == NANOCBOR_TAG_DEC_FRAC) {
+        nanocbor_value_t arr;
+        if (nanocbor_enter_array(cvalue, &arr) >= 0) {
+            res = nanocbor_get_int32(&arr, e);
+            if (res >= 0) {
+                res = nanocbor_get_int32(&arr, m);
+                if (res >= 0) {
+                    res = NANOCBOR_OK;
+                }
+            }
+        }
```

Comment:
If you properly close the container again here, the `cvalue` is advanced to after the container and can continue decoding items after the decimal fraction object. Currently the user would have to manually advance the `nanocbor_value_t` in a structure such as:

```
[
    -5,
    4([-2, 27315]),
    4([-8, 3214]),
]
```